freeipa/src/main/resources/application.yml

```diff
@@ -1,5 +1,7 @@
 server.servlet.context-path: "/freeipa"
 
+server.port: 8090
```

Comment:
Don't forget to update the cloudbreak-deployer and the kubernetes helm charts as well

Reply:
it needs modification in deployer for sure